mantidproject/mantid
# Updates to Basic Course and Error reporter testing

**Description of work.**
Changes described in the 2nd part of https://github.com/mantidproject/mantid/issues/33261

**To test:**

- build `docs-html` target and `dev-docs-html` target
- check pages and images that have changed, eg the formatting and links to pages on `docs/source/tutorials/mantid_basic_course/connecting_data_to_instruments/index.rst`

Fixes #33261 

<!-- delete this if you added release notes
*This does not require release notes* because **docs update**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
